Sources/MapboxCoreNavigation/EHorizon/LocalizedRoadObjectName.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+import MapboxNavigationNative
+
+/// Road object information, like interchange name.
+public struct LocalizedRoadObjectName: Equatable {
+    /// The name of the road object.
+    public let language: String
```

Comment:
Should this property be 'languageCode' then, like it is in Foundation?

Reply:
I was looking at `language` in [RoadName](https://github.com/mapbox/mapbox-navigation-ios/blob/8d1063e0481fc7d2e9d77aa27a6d737929551d25/Sources/MapboxCoreNavigation/EHorizon/RoadName.swift#L19) also in EV.
I think I can`languageCode` in a new struct and after think about `RoadName`

Reply:
After discussion we decided to use `language` for the consistency inside the SDK